qark/report.py

```diff
@@ -28,21 +28,21 @@ class Report(object):
     # http://python-3-patterns-idioms-test.readthedocs.io/en/latest/Singleton.html#the-singleton
     __instance = None
 
-    def __new__(cls, report_path=None):
+    def __new__(cls, issues=None, report_path=None):
```

Comment:
Issues is never used

Reply:
The `__new__` function should match the `__init__` function in terms of the parameters it accepts. `__new__` should call `__init__` after executing which will set the issues.